sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -492,6 +522,28 @@ abstract class ScrollableWidgetListState<T extends ScrollableWidgetList> extends
     ));
   }
 
+  void _setScrollOffset(double newScrollOffset) {
+    super._setScrollOffset(newScrollOffset);
+    config.scrollbarPainter?.scrollOffset = scrollOffset;
+  }
+
+  bool _scrollUnderway = false;
+
+  void _handleDragUpdate(double delta) {
+    super._handleDragUpdate(delta);
+    if (!_scrollUnderway) {
+      _scrollUnderway = true;
+      config.scrollbarPainter?.scrollStarted();
+    }
```

Comment:
We should add an onScrollStart callback and a dispatchScrollStart function so that clients can hook into this too.

Also, this should be triggered by onDragStart, not onDragUpdate.